CHANGELOG.md

```diff
@@ -197,6 +197,7 @@
 
 ### New features
 
+* [#7384](https://github.com/rubocop-hq/rubocop/pull/7384): Add new `Style/DisableCopsWithinSourceCodeDirective` cop. ([@egze][])
```

Comment:
This line needs to be moved up to the `master (unreleased)` section.

Reply:
Done

Reply:
Sorry, I should have been more clear. It should actually go under "New features" and you can put it last in the list of (unreleased) new features.

Reply:
Got it :) done